tests/unit/ceed/test_ceed.cpp

```diff
@@ -0,0 +1,179 @@
+   case Problem::Diffusion:
+      return "Diffusion";
+      break;
+   case Problem::VectorMass:
+      return "VectorMass";
+      break;
+   case Problem::VectorDiffusion:
+      return "VectorDiffusion";
+      break;
+   }
+}
+
+void test_ceed_operator(const char* input, int order, const CeedCoeff coeff_type,
+                        const Problem pb, const AssemblyLevel assembly)
+{
+   std::string section = "assembly: " + getString(assembly) + "\n" +
+                         "coeff_type: " + getString(coeff_type) + "\n" +
+                         "pb: " + getString(pb) + "\n" +
+                         "order: " + std::to_string(order) + "\n" +
+                         "mesh: " + input;
+   INFO(section);
+   Mesh mesh(input, 1, 1);
+   mesh.EnsureNodes();
+   int dim = mesh.Dimension();
+
+   H1_FECollection fec(order, dim);
+   bool vecOp = pb == Problem::VectorMass || pb == Problem::VectorDiffusion;
+   const int vdim = vecOp ? dim : 1;
+   FiniteElementSpace fes(&mesh, &fec, vdim);
+
+   BilinearForm k_test(&fes);
+   BilinearForm k_ref(&fes);
+
+   FiniteElementSpace coeff_fes(&mesh, &fec);
+   GridFunction gf(&coeff_fes);
+   FunctionCoefficient f_coeff(coeff_function);
+
+   Coefficient *coeff = nullptr;
+   switch (coeff_type)
+   {
+      case CeedCoeff::Const:
+         coeff = new ConstantCoefficient(1.0);
+         break;
+      case CeedCoeff::Grid:
+         gf.ProjectCoefficient(f_coeff);
+         coeff = new GridFunctionCoefficient(&gf);
+         break;
+      case CeedCoeff::Quad:
+         coeff = &f_coeff;
+         break;
+   }
+
+   switch (pb)
+   {
+   case Problem::Mass:
+      k_ref.AddDomainIntegrator(new MassIntegrator(*coeff));
+      k_test.AddDomainIntegrator(new MassIntegrator(*coeff));
+      break;
+   case Problem::Diffusion:
+      k_ref.AddDomainIntegrator(new DiffusionIntegrator(*coeff));
+      k_test.AddDomainIntegrator(new DiffusionIntegrator(*coeff));
+      break;
+   case Problem::VectorMass:
+      k_ref.AddDomainIntegrator(new VectorMassIntegrator(*coeff));
+      k_test.AddDomainIntegrator(new VectorMassIntegrator(*coeff));
+      break;
+   case Problem::VectorDiffusion:
+      k_ref.AddDomainIntegrator(new VectorDiffusionIntegrator(*coeff));
+      k_test.AddDomainIntegrator(new VectorDiffusionIntegrator(*coeff));
+      break;
+   }
+
+   k_ref.Assemble();
+   k_ref.Finalize();
+
+   k_test.SetAssemblyLevel(assembly);
+   k_test.Assemble();
+
+   GridFunction x(&fes), y_ref(&fes), y_test(&fes);
+
+   x.Randomize(1);
+
+   k_ref.Mult(x,y_ref);
+   k_test.Mult(x,y_test);
+
+   y_test -= y_ref;
+
+   REQUIRE(y_test.Norml2() < 1.e-12);
+}
+
+TEST_CASE("CEED", "[CEED]")
+{
+   auto assembly = GENERATE(AssemblyLevel::PARTIAL,AssemblyLevel::NONE);
+   auto coeff_type = GENERATE(CeedCoeff::Const,CeedCoeff::Grid,CeedCoeff::Quad);
+   auto pb = GENERATE(Problem::Mass,Problem::Diffusion,
+                      Problem::VectorMass,Problem::VectorDiffusion);
+   auto order = GENERATE(1,2,4);
```

Comment:
```suggestion
   auto order = GENERATE(1,2,3);
```

Order 4 is pretty slow. Maybe we should use 3 instead? Or is there a specific reason to test 4?